extension/content_script.js

```diff
@@ -502,10 +469,10 @@ function oneTimeSetup() {
 	let readableDomEls = window.parseDocument();
 	doc = new Doc(readableDomEls);
 	// If page is not readable, stop setting up the rest of the app.
-	if (doc.sentences.length === 0) {
-		debug("Stopping app init because page is not readable");
-		return;
-	}
+	// if (doc.sentences.length === 0) {
+	// 	debug("Stopping app init because page is not readable");
+	// 	return;
+	// }
```

Comment:
Temporarily commented out; doesn't work bc doc isn't ready (async problems) by the time this is called